src/primer.tex

```diff
@@ -119,12 +119,12 @@ \section{Bitcoin and Other Decentralized Protocols}
 
 One goal of the early Internet (ARPAnet) was to create a communications network
 that could withstand nuclear war~\cite{arpanet_and_nuclear_war}.
-To meet that goal, the designers removed the need for a central owner or
+To meet this goal, the designers removed the need for a central owner or
 controller, a single point of failure.
-The core Internet protocols are decentralized.
-Many other Internet-based protocols are similarly decentralized,
+The core internet protocols are decentralized.
```

Comment:
Apparently, the question of whether Internet should be capitalized is still a matter of debate unless you're inside an organization with an official position on the matter. For example, see https://blog.oxforddictionaries.com/2016/04/should-you-capitalize-internet/